modules/pom.xml

```diff
@@ -314,7 +315,7 @@
                 <jdk>[9,)</jdk>
             </activation>
             <properties>
-                <patch-gen.system.args>--add-opens=java.base/java.lang=ALL-UNNAMED</patch-gen.system.args>
+                <patch-gen.jvm.args.add-opens>--add-opens=java.base/java.lang=ALL-UNNAMED</patch-gen.jvm.args.add-opens>
```

Comment:
Looks like this is never used? You're probably missing a default value for `patch-gen.jvm.args`.